includes/razorpay-webhook.php

```diff
@@ -54,28 +54,45 @@ public function process()
                     write_log(['message' => $e->getMessage()]);
                     return;
                 }
-            }
 
-            switch ($data['event'])
-            {
-                case 'payment.authorized':
-                    return $this->paymentAuthorized($data);
+                switch ($data['event'])
```

Comment:
@ccrims0n We should have, but for now released a fix with webhooks disabled by default. This PR will likely get merged today / tomorrow.